Formula/muffet.rb

```diff
@@ -0,0 +1,22 @@
+class Muffet < Formula
+  desc "Fast website link checker in Go"
+  homepage "https://github.com/raviqqe/muffet"
+  url "https://github.com/raviqqe/muffet/archive/v2.3.0.tar.gz"
+  sha256 "6a84c058054586eadae3a0cdfab1ffe173e44f2eadce91d2a7e928018a340682"
+  license "MIT"
+  head "https://github.com/raviqqe/muffet.git"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    assert_match "failed to fetch root page: lookup does.not.exist: no such host",
+        shell_output("#{bin}/muffet https://does.not.exist 2>&1", 1)
+
+    system bin/"muffet", "https://httpbin.org/"
+    assert_equal 0, $CHILD_STATUS.exitstatus
```

Comment:
Is this line necessary tho? I think the test fails automatically when system call exits with non zero code.

Reply:
i grab it from other place, i can take it off though. :)